docs/diagrams/DeletePatientSequenceDiagram.puml

```diff
@@ -49,7 +49,7 @@ deactivate AddressBookParser
 LogicManager -> DeleteCommand : execute(m)
 activate DeleteCommand
 
-DeleteCommand -> Model : deletePerson(1)
+DeleteCommand -> Model : deletePerson(patient)
```

Comment:
Can consider to add the `Person patientToDelete = model.getFilteredPatientById(allPersons, patientId)` interaction between the DeletePatientCommand and the model before the deletePerson() interaction is shown as it seems that the `patient` is out of nowhere